src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String descr, String by) {
+        super(descr);
```

Comment:
I think that lengthening 'descr' to 'description' would be better, to make it more clear